docs/content/preview/yugabyte-platform/troubleshoot/install-upgrade-issues/kubernetes.md

```diff
@@ -0,0 +1,279 @@
+
+**Symptom**
+
+There can be multiple reasons behind `CrashLoopBackOff`. YBA pod can crash due to some internal application error.
+
+**Debug**
+
+You can use `kubectl` to analyze the situation. We need to check YBA logs to understand the failure.
+
+- Get all pods with their status in a namespace.
+
+  ```sh
+  kubectl get pod -n <NAMESPACE>
+
+  ## output
+  ## NAME                             READY   STATUS    RESTARTS   AGE
+  ## yugabyte-platform-1-yugaware-0   4/4     Running   0          4d14h
+  ```
+
+- Describe the YBA pod facing `CrashLoopBackOff`. Check for `Events`.
+
+  ```sh
+  kubectl describe pods <YBA-POD-NAME> -n <NAMESPACE>
+  ```
+
+- Check the logs for particular container
+
+  ```sh
+  ## yugabytedb anywhere
+  kubectl logs <YBA-POD-NAME> -n <NAMESPACE> -c yugaware
+
+  ## postgres
+  kubectl logs <YBA-POD-NAME> -n <NAMESPACE> -c postgres
+  ```
+
+## CORS error while accessing YBA through LB
+
+You might experience the issue while accessing the YugabyteDB Anywhere through a load balancer. You can define the Cross-Origin Resource Sharing (CORS) domain configuration by setting the [additionAllowedCorsOrigins](https://github.com/yugabyte/charts/blob/master/stable/yugaware/values.yaml#L66) value to the new domain involved. For example, you would add the following to the appropriate Helm command.
+
+  ```properties
+   --set additionAllowedCorsOrigins:'https://mylbdomain'
+  ```
+
+## Load Balancer + EKS
+
+Sometimes the default Amazon Web Services (AWS) load balancer brought up in Amazon Elastic Kubernetes Service (EKS) by the YugabyteDB Anywhere Helm chart is not suitable for your setup, you can use the following settings to customize the [AWS load balancer controller](https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.2/guide/service/annotations/) behavior.
+  - `aws-load-balancer-scheme` can be set to `internal` or `internet-facing` string value.
+  - `aws-load-balancer-backend-protocol` and `aws-load-balancer-healthcheck-protocol` should be set to the `http` string value.
+
+  Consider the following sample configuration:
+
+  ```properties
+  service.beta.kubernetes.io/aws-load-balancer-type: "ip"
+  service.beta.kubernetes.io/aws-load-balancer-scheme: "internet-facing"
+  service.beta.kubernetes.io/aws-load-balancer-backend-protocol: "http"
+  service.beta.kubernetes.io/aws-load-balancer-healthcheck-protocol: "http"
+  ```
+
+## Is the load balancer up?
+
+### Is a load balancer controller avail?
+
+Most cloud providers provide the load balancer controller to serve the load balancer type service. You need to install the controller manually in case of its absence, or else the load balancer service wouldn't work as intended. For GKE, see [GKE Ingress](https://cloud.google.com/kubernetes-engine/docs/concepts/ingress#overview).
+
+### Public IP quota
+
+You might experience Public IP not associated with the load balancer if you exceed the Public IP quota limit.
+
+## Unable to expand PVC
```

Comment:
This is not supported for YBA at least. So what we could show is the error which helm gives when we change the volume size, and say that currently it is not possible to expand the volume via a helm operation.
Or it is possible in YBA? Because we create a PVC separately?

Reply:
Agree - if we cannot expand this PVC without recreating the statefulset, we should just say that. Does simply changing the volume size work? If just changing the helm value doesn't work, we can say something like

```
In general, resizing a storage volume created through a statefulset is challenging and not recommended. In cases where no other option exists, please follow steps similar to those documented at https://www.howtogeek.com/devops/how-to-resize-a-kubernetes-statefulsets-volumes/ to resize the storage volume and recreate the statefulset.
```